app/styles/templates/_app_menu_list.scss

```diff
@@ -0,0 +1,44 @@
+.app-menu-list {
+  .center {
+    text-align: center;
+  }
+
+  .top-menu {
+    display: flex;
+  }
+
+  .menu-section {
+    background-color: #8091A9;
```

Comment:
use variable `$tabbar-menu-icon-color` (present in app/styles/_settings.scss)

Reply:
Done!